src/interface.ts

```diff
@@ -11,7 +11,20 @@ export * from './components/dialog/dialog.types';
 export * from './components/dock/dock.types';
 export * from './components/file/file.types';
 export * from './components/flex-container/flex-container.types';
-export * from './components/form/form.types';
+export {
+    FormComponent,
+    FormComponentOptions,
+    FormInfo,
+    FormLayoutOptions,
+    FormLayoutType,
+    FormSchemaArrayItem,
+    FormSubKeySchema,
+    GridLayoutOptions,
+    RowLayoutOptions,
+    LimeLayoutOptions,
+    LimeSchemaOptions,
+    ReplaceObjectType,
+} from './components/form/form.types';
```

Comment:
Is there something from here we don't want to export? I could be easy to forget to add new types to this list in the future. Perhaps the type we don't want to export could be moved to another file?

Reply:
Yeah, Maybe we can skip ALL the utility types, that is used just for the generic stuff.

Reply:
I have fixed this in a fixup commit 👍 
But as before I cannot run the api-update so you have to do it for me 👎

Reply:
Those "utility" types are part of public types, so they need to be public too. So I've moved them back into the `form.types.ts` and changed the export here to just export everything from it.

Reply:
Thanks, nice!